### PR TITLE
fix(build): included management library into the build

### PR DESCRIPTION
**Pull request by bluestreak01**
We found out that `dump_thread_stacks()` does not work on docker and `RT` builds. This was because we didn't package the management library in the build.

I also added "enjoy" message to easily identify restarts in the log

### CODE REVIEW

**Review by puzpuzpuz — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/ServerMain.java|0|1|00.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/ServerMain.java|0|1|00.00%|

File: core/src/main/java/io/questdb/ServerMain.java [hunk 1]
```diff
@@ -211,6 +211,8 @@ public ServerMain(String[] args) throws Exception {
 
             System.gc();
 
+            log.advisory().$("enjoy").$();
```
Comment (rstreics) [inline core/src/main/java/io/questdb/ServerMain.java:214]: I would suggest we go with "QuestDB Instance restarted" or something similar. In long run "enjoy" in logs would confusing.

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/ServerMain.java|0|1|00.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/ServerMain.java|0|1|00.00%|

File: core/src/main/java/io/questdb/ServerMain.java (hunk 1 above)
Comment (bluestreak01) [reply core/src/main/java/io/questdb/ServerMain.java:214]: "enjoy" is a very unlikely word to be used in technical logging, in fact this idea comes from a large production cluster of applications I used to work on. The benefits are:
- easy to memorize
- low probability of false positive

We could get used to using "enjoy" in context of "instance started".

I would argue that `"QuestDB Instance restarted` is harder to memorize: "(I)nstance" - capital 'I', 'started" or "restarted" ? etc.

**Pull request merged**